tests/gitops-ztp/internal/params/params.go

```diff
@@ -0,0 +1,35 @@
+package params
+
+import "time"
+
+type ArgocdGitDetails struct {
+	Repo   string
+	Branch string
+	Path   string
+}
+
+const (
```

Comment:
Please check https://github.com/openshift-kni/eco-gosystem/blob/main/tests/ran-du/internal/randuconfig/config.go if you want to make these parameters configurable via a yaml file.